plugin/efficientNMSPlugin/efficientNMSInference.cu

```diff
@@ -314,12 +314,16 @@ __global__ void EfficientNMS(EfficientNMSParameters param, const int* topNumData
 
         for (int tile = 0; tile < numTiles; tile++)
         {
+            bool check_class = true;
+            if (!param.classAgnostic)
+               check_class = threadClass[tile] == testClass;
```

Comment:
With this logic, `check_class` would be true if the `class_agnostic` flag is set. 
```suggestion
            bool check_class = false;
            if (!param.classAgnostic)
               check_class = threadClass[tile] == testClass;
```

Reply:
When `param.classAgnostic == true`, there is no need to do class check. So the default value of  check_class is true. I think my code is confusing,  maybe the code below is better?
`
bool check_class;
if (param.classAgnostic)
    check_class = true;
else
    check_class = threadClass[tile] == testClass;
`